docs/ide/package/package.md

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
I don't think this new page is necessary.  IMO an overview of packaging technologies in VS doesn't need to be included in the installer projects docs.  It would also be hard to get this page right, for example you'd have to mention technologies like ClickOnce, which certainly falls under the umbrella of "tools for packaging desktop applications for Windows".